extensions/ql-vscode/src/view/common/CodePaths/CodePaths.tsx

```diff
@@ -24,6 +24,11 @@ export type CodePathsProps = {
   severity: ResultSeverity;
 };
 
+type ScollPosition = {
```

Comment:
```suggestion
type ScrollPosition = {
```
hey, just here to dome some open source, little typo :)